test_helpers/main.go

```diff
@@ -96,7 +96,7 @@ func isReady(dialer tarantool.Dialer, opts *tarantool.Opts) error {
 	}
 	defer conn.Close()
 
-	resp, err = conn.Do(tarantool.NewPingRequest()).Get()
+	resp, err = conn.Do(tarantool.NewPingRequest()).GetResponse()
```

Comment:
And what will happen in case I use `Get()`? Segmentation fault? What's the valid reasons for user to call `GetResponce()` instead of `Get()`?

Reply:
No, no panics or anything like that. `Get` will return correct response data (obtained after `GetResponse` + `Decode`) or nil + error.
In the first commit, using `GetResponse` we can obtain an actual response and, for example, call `Pos`, `MetaData` or `SQLInfo` methods. This is also true for the second and third commits, but also we could obtain a `Header` part of the response. With this we can get a `ResponseId` and an `Error` (and using `Error` we can recognize invalid response without decoding it).